src/preferences/dialog/dlgprefbroadcast.cpp

```diff
@@ -145,6 +145,14 @@ DlgPrefBroadcast::DlgPrefBroadcast(QWidget *parent,
      comboBoxEncodingChannels->addItem(tr("Stereo"),
              static_cast<int>(EncoderSettings::ChannelMode::STEREO));
 
+     // Encryption mode combobox
+     encryptionModeComboBox->addItem(tr("Required"),
+             static_cast<int>(BroadcastProfile::EncryptionMode::Required));
+     encryptionModeComboBox->addItem(tr("Preferred"),
+             static_cast<int>(BroadcastProfile::EncryptionMode::Preferred));
+     encryptionModeComboBox->addItem(tr("Disabled"),
+             static_cast<int>(BroadcastProfile::EncryptionMode::Disabled));
```

Comment:
I consider "Preferred" as a security risk. In case encryption fails, it should never fall back to plain. 

We may add a mode like "Try encryption" in case it works, it should be stored as "Required" and fail later if the server is no longer the verified pair.

Reply:
Yeah, that's a good point. I'm not sure how feasible this is tho, without requirement too much refactor. I might disable `Preferred` for now to allow us moving forward, wdyt?

Reply:
That works for me.